package sv_cl_messages;

message ServerInfo
{
	required string ipAddress = 1;
	required string hostName = 2;
}